packages/node_modules/@ciscospark/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -122,21 +122,60 @@ export const transforms = toArray('inbound', {
   },
 
   /**
-   * Decrypts a content field which contains files
+   * Decrypts a content field
    * @param   {Object} ctx            An object containing a spark instance and transform prop
    * @param   {String} key            KMS key
    * @param   {Object} content        An object with properties to be decrypted
    * @parma   {Array}  content.files  An array of files to decrypt by calling decryptObject
    * @returns {[type]}                [description]
    */
   decryptContent(ctx, key, content) {
+    if (content.contentCategory === 'links') {
+      return ctx.transform('decryptContentLinks', key, content);
+    }
+
+    return ctx.transform('decryptContentFiles', key, content);
+  },
+
+  /**
+   * Decrypts a content field which contains files and possibly links
```

Comment:
Regarding "possibly links", isn't this determined by the `content.contentCategory === 'links'` above? Or are there other instances when the `contentCategory` is not `links` but there are still links?

Reply:
All the other contentCategories can have links. This way a person can send a document and a link. If it is only links, the contentCategory will be 'link'